app/src/main/java/de/westnordost/streetcomplete/quests/internet_access/InternetAccess.kt

```diff
@@ -0,0 +1,8 @@
+package de.westnordost.streetcomplete.quests.internet_access
+
+enum class InternetAccess(val osmValue: String) {
+    WLAN("wlan"),
```

Comment:
should rename the enum to "WIFI", WLAN is only really used in German.
The OSM value must stay "wlan" of course.